Makefile

```diff
@@ -0,0 +1,97 @@
+version=3.81
+
+PROJECT_DIR ?= .
+DOCKER_IMAGE_TAG ?= dev
+DOCKER_REGISTRY := ghcr.io
+DOCKER_ORG := darpa-askem
+
+
+
+.SECONDEXPANSION:
+
+.PHONY: all
+all:
+	@echo "make <cmd>"
+	@echo ""
+	@echo "Commands:"
+	@echo "  clean                    - remove bin, dist and node_modules directories"
+	@echo "  images                   - build docker images of all targets"
+	@echo "  image-<target>           - build docker image of a specific target - see below for list of targets"
+	@echo "  publish                  - build and push dock images for deployment"
+	@echo "  publish-<target>         - build and push dock image for a specific target - see below for list of targets"
+	@echo "  lint                     - lint the source code"
+	@echo "  test                     - test the source code"
+	@echo "Targets building (but not pushing) docker images of separately:"
+	@echo "  $(TARGETS)"
+	@echo ""
+	@echo "Variables:"
+	@echo "  DOCKER_IMAGE_TAG=<tag>         - tag to append to each pushed docker image (default: dev)"
+
+
+
+## Targets
+TARGETS :=
+
+TARGETS += hmi-server
+clean-hmi-server:
+	./gradlew :packages:services:hmi-server:clean
+
+image-hmi-server: clean-hmi-server
+	./gradlew :packages:services:hmi-server:build -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=$(DOCKER_IMAGE_TAG)
```

Comment:
where is it getting the org and name from? ie ghcr.io etc..

Reply:
hmm.. okay may need to rework this a bit
again the focus of the PR is that it builds not how the images are built, that we can push to another PR

Reply:
please make a new ticket